Cubical/Algebra/Ring.agda

```diff
@@ -1,7 +1,5 @@
-{-# OPTIONS --safe #-}
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
 module Cubical.Algebra.Ring where
 
 open import Cubical.Algebra.Ring.Base public
```

Comment:
Can you please not have these imports public?

Reply:
Are you sure? As far as I understand it, files like 'Cubical.Algebra.Ring.agda' will be useless then.

Reply:
Oops, I was confused. Please ignore this